ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -761,7 +761,7 @@ export default class ConfirmTransactionBase extends Component {
                   });
                 },
                 onDeepLinkShown: () => {
-                  this.props.clearConfirmTransaction();
+                  clearConfirmTransaction();
```

Comment:
Was this change for code style or function?

Reply:
Code style, we're destructuring the props, some lines above, so I want to maintain everything the same.